backends/graphs/parser.cpp

```diff
@@ -149,16 +149,21 @@ bool ParserGraphs::preorder(const IR::PackageBlock *block) {
             pGraph g_;
```

Comment:
shouldn't you be clearing `acceptParents` and `rejectParents` at each iteration of the for loop?
It seems that if you have a P4 architecture with several parser blocks, you would not be resetting these data structures properly between each iteration.

Reply:
agree, fixed.